### ISSUE TITLE
nx run shared-ui:storybook fails after NX upgrade to @nrwl/storybook : 13.3.0-beta.8 & Webpack  5

### ISSUE THREAD

**Issue by codeninja** [nrwl/nx#7994]
## Current Behavior

![image](https://user-images.githubusercontent.com/14914/144660371-c1e08146-91af-42cc-85f6-63b26c1b0680.png)

## Expected Behavior

Storybook builds and serves.

## Steps to Reproduce

Our main branch:
![image](https://user-images.githubusercontent.com/14914/144661357-5a62447d-8755-422d-8c75-9dc62fedb185.png)

Our upgrade branch:
```
nx upgrade next
```
![image](https://user-images.githubusercontent.com/14914/144661379-8b9fdcd3-7b09-4d18-8361-edadbadcb6d9.png)

All things storybook :
![image](https://user-images.githubusercontent.com/14914/144661885-a41b0379-ad02-4d0e-9656-b2f7e80bd856.png)

This issue may not be prioritized if details are not provided to help us reproduce the issue.

**Comment by codeninja** [comment]
FYI: For anyone else who runs across this, I fixed it by removing @storybook/react and re-installing it. Somehow @storybook/react was including @storybook/core-common from an older version and webpack picked it up and brought it in.

TLDR: Nuke your packages and install @storybook fresh.
